src/videoCache.js

```diff
@@ -162,6 +175,20 @@ export function getCacheUrl(id) {
   return `${config.getConfig('cache.url')}?uuid=${id}`;
 }
 
+export const storeLocally = (bid) => {
+  const vastValue = getVastXml(bid);
+  const strategy = config.getConfig('cache.strategy') || BLOB;
```

Comment:
Why expose these / make them mutually exclusive? I'd always generate a blob and translate it to a string only if/when we need to. It's less for the pub to understand and it also allows using different players with the same bid.